web/src/WirelessSelector.jsx

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright (c) [2022] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React, { useState } from "react";
+import Popup from "./Popup";
+import {
+  Alert,
+  Button,
+  Card,
+  CardBody,
+  Form,
+  FormGroup,
+  FormSelect,
+  FormSelectOption,
+  Radio,
+  TextInput
+} from "@patternfly/react-core";
+
+import {
+  EOS_LOCK as LockIcon,
+  EOS_SIGNAL_CELLULAR_ALT as SignalIcon
+} from "eos-icons-react";
+
+import Center from "./Center";
+
+import { useInstallerClient } from "./context/installer";
+
+const CONNECTION_FORM_ID = "chosen-network";
+const baseHiddenNetwork = { ssid: "", hidden: true };
+
+function WirelessConnectionForm({ network, setSubmittingData, onClose }) {
+  const client = useInstallerClient();
+  const [error, setError] = useState(false);
+  const [ssid, setSsid] = useState(network.ssid);
+  const [password, setPassword] = useState("");
+  const [security, setSecurity] = useState("none");
+
+  const security_options = [
+    { value: "", label: "None" },
+    { value: "wpa-psk", label: "WPA & WPA2 Personal" },
+    { value: "wpa-eap", label: "WPA & WPA2 Enterprise" }
+  ];
+
+  const selectorOptions = security_options.map(security => (
+    <FormSelectOption key={security.value} value={security.value} label={security.label} />
+  ));
+
+  const connectNetwork = async () => {
+    await client.network.connectTo(ssid, { security, password });
```

Comment:
@teclator,

At this point, we have to send the hidden option when needed, right?